opencti-platform/opencti-front/src/private/components/entities/organizations/OrganizationKnowledge.js

```diff
@@ -77,6 +77,28 @@ class OrganizationKnowledgeComponent extends Component {
             ))
             }
           />
+          <Route
+            exact
+            path="/dashboard/entities/organizations/:organizationId/knowledge/targets"
+            render={(routeProps) => (
+              <EntityStixCoreRelationships
+                entityId={organization.id}
+                relationshipTypes={['targets']}
+                isRelationReversed
+                entityLink={link}
+                allDirections={true}
+                stixCoreObjectTypes={[
```

Comment:
According to the stix.ts file, it's missing the type Attack-Pattern.

Reply:
Fixed